src/App.js

```diff
@@ -1,13 +1,55 @@
-import React from 'react';
-
+import React, { PureComponent } from 'react';
+import GoodsList from './components/GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
 import './App.scss';
+import Button from './components/Button';
+// eslint-disable-next-line max-len
+
+class App extends PureComponent {
+  state = {
+    goodsToRender: null,
+  }
+
+  renderGoods = () => {
+    getAll().then(goods => this.setState({ goodsToRender: goods }));
+  }
+
+  getRedsGoods = () => {
+    getRedGoods().then(goods => this.setState({ goodsToRender: goods }));
+  }
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+  getSortedGoods = () => {
+    get5First().then(goods => this.setState({ goodsToRender: goods }));
+  }
```

Comment:
let's create ONE function which will accept a callback (getAll, getRedGoods or get5First) and subscribe to its result

Reply:
we can it into the Button and import all api functions there